### ISSUE TITLE
NotificationDrawer: Add 'x' to drawer to help understand how to close an open drawer

### ISSUE THREAD

**Issue by jcaianirh** [patternfly/patternfly#3072]
There is a current openshift bug that is requesting an 'x' be added to the notification drawer so that the user understands how to close it.

https://bugzilla.redhat.com/show_bug.cgi?id=1800614

**Comment by mcoker** [comment]
cc @mcarrano

**Comment by jcaianirh** [comment]
@mcoker @mcarrano any updates here?

**Comment by mcarrano** [comment]
Yeah, this was probably an oversight more than anything else. I will transfer over to design as an enhancement request and work to get this on our roadmap.
